packages/react-icons/README.md

```diff
@@ -0,0 +1,23 @@
+# @coveord/plasma-react-icons
+
+This package exposes the Plasma design system's entire iconography in the shape of individual React components.
```

Comment:
```suggestion
This package exposes the Plasma design system entire iconography in the shape of individual React components.
```

Reply:
I disagree

Reply:
I invoke @kdang-coveo for the final say :eyes: